package.json

```diff
@@ -13,7 +13,8 @@
     "eslint:check": "eslint -c ./.eslintrc.cjs --max-warnings 0 ./",
     "prettier:check": "prettier -c --no-config ./",
     "check": "npm run eslint:check && npm run prettier:check",
-    "prettier:fix": "prettier -w --no-config ./"
+    "prettier:fix": "prettier -w --no-config ./",
+    "vscode:dev": "cd vscode && npx serve --cors -l 5000 --ssl-cert $HOME/certs/localhost.pem --ssl-key $HOME/certs/localhost-key.pem"
```

Comment:
This won't work on Windows. Also, if we're keeping it, I think this should go in `vscode/package.json` along with the other command for launching VS Code.

Reply:
It's also not entirely clear to me what advantage this has over the `run:web` command we already have in the other package.json?

`"run:web": "npx @vscode/test-web --extensionDevelopmentPath . ../samples"`

Reply:
It allows one to test in vscode.dev which is different than the web ext host. For windows, use WSL and this will work if you set up the environment. For which file it goes in, I'm really tired of moving into the vscode directory to run tasks. Do you have any suggestions?

Reply:
`npm -w vscode run ___` at the top level will let you run a script from `vscode/package.json` . 

I'm not opposed to adding tasks to the top-level package.json for convenience, I just think if we're going to do that, we should move the existing tasks up there too. Not a fan of having one-off scripts sprinkled across different files.

The script should be usable on Windows without WSL - more than a couple devs use Windows on the team, and our other build scripts account for this.

Maybe this can just go in your personal `tasks.json`?

Reply:
> vscode.dev which is different than the web ext host

I'm not sure about this - doesn't vscode/test-web serve a is a locally hosted vscode.dev ? I think they're the same app.

Reply:
Yes, I believe they're the same app.

Regarding the task, besides not working on Windows, is has options such as `--ssl-cert $HOME/certs/localhost.pem --ssl-key $HOME/certs/localhost-key.pem"`, so that's not going to work unless you do some prior setup to create certs in the correct location and with the correct names - none of which is documented in this PR.

Tasks we check-in should work out of the box or have docs on how to use them, and work on all supported platforms.